docs/content/release_notes/limitations.md

```diff
@@ -25,7 +25,7 @@ The following limitations apply when using volume clones with the IBM block stor
 
 -   A PVC and its clone need to both have the same volume mode (**Filesystem** or **Block**).
 
-**Note:** For volume clone limitations pertaining specifically to HyperSwap usage, see [HyperSwap usage limitations](#hyperSwap-usage-limitations).
+**Note:** For high availability volume clone limitations, see [High availability limitations](#high-availability-lilimtations).
```

Comment:
please go over the diff here and verify that the new links actually work.
@matancarmeli7 could you please ask to prioritize CSI-3648 in the next sprint?
this type of check should be made by GitHub, not by reviewers

Reply:
done

Reply:
thanks.
@rivkap let me know when you finish checking the links, so I can resolve this

Reply:
done and updated

Reply:
I don't see a change in this line

Reply:
please verify your changes using:
![image](https://user-images.githubusercontent.com/57443811/155884662-461cb71f-b9e5-4454-accd-a8edbb03f13a.png)

Reply:
this line specifically doesn't require a change. both hyperswap and stretched have these limitations

Reply:
@rivkap this conversation is not about linking to more precise locations, it's about not merging broken links to develop branch.
as I wrote in the first comment of this conversation, please ensure all links in this PR diff actually work.
e.g. the link in this line still doesn't work for me.
I try my best to cover all link changes when I check the diff, it would be extremely helpful if you always do the same **before** I do.
let me know if you need any clarifications about how to perform these checks.

Reply:
I checked all of the links on this PR and updated any that needed updating

Reply:
the link here still doesn't work for me.
let me know when you want me to show you how to check a link

Reply:
I had checked all of the links but must have missed this one.
I fixed it now and it should work 
![image](https://user-images.githubusercontent.com/54570654/156325994-48dce301-b987-423d-913e-9ff3c1518710.png)